monkey/monkey_island/cc/services/node.py

```diff
@@ -261,6 +267,7 @@ def get_monkey_island_pseudo_net_node():
     def get_monkey_island_node():
         island_node = NodeService.get_monkey_island_pseudo_net_node()
         island_node["ip_addresses"] = local_ip_addresses()
+        island_node["domain_name"] = ""
```

Comment:
Why no domain_name if we know it's the island? can't we query the server hostname?